skbio/alignment/_tabular_msa.py

```diff
@@ -685,13 +674,14 @@ def reindex(self, key=None, keys=None):
         array(['a', 'b'], dtype=object)
 
         """
-        if key is not None and keys is not None:
+        if minter is not None and keys is not None:
             raise ValueError(
-                "Cannot use both `key` and `keys` at the same time.")
+                "Cannot use both `minter` and `keys` at the same time.")
 
         keys_ = None
-        if key is not None:
-            keys_ = [resolve_key(seq, key) for seq in self._seqs]
+        if minter is not None:
```

Comment:
I've implemented this. One side affect of the second bullet is that deleting the keys also deletes the minter. This makes sense, right? Also see [my comment here](https://github.com/biocore/scikit-bio/pull/1121#discussion_r41322924)